app/src/main/java/org/fossasia/openevent/general/AuthActivity.kt

```diff
@@ -68,4 +74,13 @@ class AuthActivity : AppCompatActivity() {
         super.onBackPressed()
         startActivity(Intent(this, MainActivity::class.java).setFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP))
     }
+
+    private fun isSnackbar() {
+        showSnackbar = intent.getStringExtra("showSnackbar")?:"false"
+        if (showSnackbar == "true") {
```

Comment:
That might be a first for me in 6 years. May I ask why this brilliant idea? /s

I asked you to pass the string to be shown in snackbar, not write "true" and "false" like booleans don't exist in the language. Try to understand why reviewers ask changes, and think how is this better from what was there before. Spoiler Alert: It is not

Reply:
Indeed I think the same way when writing above code

> Pass a string instead of boolean

I misinterpret above statement, will fix soon